RosettaCore/src/main/java/com/hubspot/rosetta/RosettaBinder.java

```diff
@@ -15,6 +16,24 @@ public interface Callback {
     void bind(String key, Object value);
   }
 
+  public void bindList(ArrayNode node, String field, Consumer<Object> valueConsumer) {
+    for (JsonNode value : node) {
+      if (!field.isEmpty()) {
+        if (!value.has(field)) {
```

Comment:
The code here is good, but it can be a little tough to wrap your head around. What do you think about 
```java
if (!field.isEmpty()) {
  value = value.path(field);
  if (value.isMissingNode()) {
    throw new IllegalArgumentException(String.format("Field %s does not exist in elements of input array", field));
  }
}
```

I think this may make what's happening here a little more clear.

Reply:
Sgtm 👍